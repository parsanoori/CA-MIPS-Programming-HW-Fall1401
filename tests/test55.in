36
724699384
440162916
467685012
95605523
547365598
617860570
348432704
410780282
878464346
57195164
313088848
413789445
954322359
713477988
554315721
624215566
806978420
84950065
573415422
45965200
678397966
492736644
819600679
1065656926
777538725
844671580
64231225
581145882
184449949
992574271
596877156
338792563
822012312
159205425
1013940064
731513354