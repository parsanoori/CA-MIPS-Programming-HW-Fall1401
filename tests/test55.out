45965200
57195164
64231225
84950065
95605523
159205425
184449949
313088848
338792563
348432704
410780282
413789445
440162916
467685012
492736644
547365598
554315721
573415422
581145882
596877156
617860570
624215566
678397966
713477988
724699384
731513354
777538725
806978420
819600679
822012312
844671580
878464346
954322359
992574271
1013940064
1065656926